### ISSUE TITLE
Completion only for references containing a PDF file

### ISSUE THREAD

**Issue by lyndhurst** [jalvesaq/zotcite#41]
Hi,
sorry if I missed something obvious in the doc, but the only references listed by the `<C-X><C-o>` completion are Zotero PDF references. Other zotero items like word, excel documents, web pages, images are ignored.
Is it the intended behaviour?

**Comment by jalvesaq** [comment]
I have a collection containing fake items of various types, and the omni completion works with all of them: journal article, book, interview, film, audio recording, letter, patent, map, report, artwork, software, presentation, bill, and podcast. The title of one item mixes Latin and Greek letters and another one has both title and author names in Chinese, and the completion works with them too.

**Comment by lyndhurst** [comment]
Thank you for such a quick reply. 

This is good to know, I am going to try and debug seriously tomorrow morning because after tinkering with the plugin for a bit, I have a lot of issues like no key mappings, citations just show the file title, no zotero id...

So I am sure this is something wrong on my side, I just need to figure out the right angle to debug my installation. Thank you again, I will close this tomorrow if I manage to fix this.

**Comment by lyndhurst** [comment]
It was not that hard to debug actually, I just had to reinstall it correctly, sorry for not trying that first.

Everything is working fine, except of course the syntax part in pandoc files as you did specify in the readme. I simply copied the content of the syntax file from your plugin in my `.vim/after/syntax/pandoc.vim` where I have a few other syntax alterations, it works like a charm.

Thank you again for your time.
